en/flight_controller/beaglebone_blue.md

```diff
@@ -17,118 +17,114 @@ Information about flashing OS images can be found on [this page](https://github.
-   ```sh
-    ...@ubuntu:/opt/bbblue_toolchain$ ls -l
-        lrwxrwxrwx 1 root root   51 Mar 22 16:10 gcc-arm-linux-gnueabihf -> gcc-linaro-6.3.1-2017.05-x86_64_arm-linux-gnueabihf
-        drwxr-xr-x 8 root root 4096 May 17  2017 gcc-linaro-6.3.1-2017.05-x86_64_arm-linux-gnueabihf
-        drwxr-xr-x 8 root root 4096 Nov 19 03:51 gcc-linaro-6.4.1-2017.11-x86_64_arm-linux-gnueabihf
-   ```
-    
-1. Add it to the PATH in ~/.profile as shown below
-   ```sh
-   export PATH=$PATH:/opt/bbblue_toolchain/gcc-arm-linux-gnueabihf/bin
-   export CrossCompiler=/opt/bbblue_toolchain/gcc-arm-linux-gnueabihf/bin/arm-linux-gnueabihf-
-   ```
+The steps are:
+```
+sudo -su
+connmanctl
+connmanctl>scan wifi
+connmanctl>services(at this point you should see your network appear along with other stuff, in my case it was "AR Crystal wifi_f45eab2f1ee1_6372797774616c_managed_psk")
+connmanctl>agent on
+connmanctl>connect wifi_f45eab2f1ee1_6372797774616c_managed_psk
+connmanctl>quit
+```
+### SSH root Login on Beaglebone
+With USB Serial connectet 
+```
+echo "PermitRootLogin yes" >>  /etc/ssh/sshd_config && systemctl restart sshd
+```
 
-> **Tip** GCC in the toolchain should be compatible with kernel in *BeagleBone Blue*. General rule of thumb is to choose a toolchain 
-where version of GCC is not higher than version of GCC which comes with the OS image on *BeagleBone Blue*. 
+### Cross Compiler Setup
 
+1. First set up *rsync* (this is is used to transfer files from the development computer to the target board over a network - WiFi or Ethernet). 
+   For *rsync* over SSH with key authentication, follow steps here:
+   1. sudo su
+   1. ssh-keygen -t rsa -C root@beaglebone
+   1. ENTER //no passphrase
+   1. ENTER
+   1. ENTER
+   1. ssh-copy-id root@beaglebone
+   1. When promted if you trust - yes
+   1. password root
+   1. On the development computer, define the BeagleBone Blue board as `beaglebone` in **/etc/hosts**
+1. Cross Compile Setup
+   1. Toolchain download
+      1. First install the toolchain into */opt/bbblue_toolchain/gcc-arm-linux-gnueabihf*.
+         Here is an example of using soft link to select which version of the toolchain you want to use:
+         ```
+         mkdir -p /opt/bbblue_toolchain/gcc-arm-linux-gnueabihf
+         chmod -R 777 /opt/bbblue_toolchain
+         ```
+         ARM Cross Compiler for *BeagleBone Blue* can be found at [Linaro Toolchain Binaries site](http://www.linaro.org/downloads/). 
+  
+   
+         > **Tip** GCC in the toolchain should be compatible with kernel in *BeagleBone Blue*.
+            General rule of thumb is to choose a toolchain where version of GCC is not higher than version of GCC which comes with the OS image on *BeagleBone Blue*. 
+   
+         Download and unpack [gcc-linaro-7.5.0-2019.12-x86_64_arm-linux-gnueabihf](https://releases.linaro.org/components/toolchain/binaries/latest-7/arm-linux-gnueabihf/gcc-linaro-7.5.0-2019.12-x86_64_arm-linux-gnueabihf.tar.xz) to the bbblue_toolchain folder
```

Comment:
Might be woth placing this above the Tip, so that people don't have to search first.

Reply:
> Thanks.
> In general can you document on which machine (host or beaglebone) to execute commands?

was hard for me to follow the actual manual, so i changed what i note for me, sorry when something is not clear

Reply:
Beat is suggesting that this line could be moved up above the Tip. Can you do that? Not sure if you're saying yes or no :-)